features/cart/shopping_cart/deleting_expired_carts_automatically.feature

```diff
@@ -0,0 +1,20 @@
+@shopping_cart
+Feature: Deleting expired carts automatically
+    In order to get rid of started but not finished orders
+    As an Administrator
+    I want to have expired carts automatically deleted
+
+    Background:
+        Given the store operates on a single channel in "United States"
+
+    @domain
+    Scenario: Having cart deleted after 3 hours of idleness
```

Comment:
:( What a horrible world would it be -> I will fix this in a separate PR together with fixes from https://github.com/Sylius/Sylius/pull/6291